src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -351,6 +363,77 @@ def get_stats_data_for_trending_events(results):
                 else {},
             }
 
+        def fingerprint_regression(trending_event):
+            return hashlib.sha1(
+                (
+                    f'p95_transaction_duration_regression-{trending_event["transaction"]}-experiment'
+                ).encode()
+            ).hexdigest()
+
+        def send_occurrence_to_plaform(found_trending_events):
```

Comment:
These functions should not be in this file. 

This PR is a hack that we are putting in to production in order to get some issues flowing, not new endpoint functionality. As such here are my recommendations:

* Put this in its own file 
* This function should not depend on  the `request` (which it does now)
* Explain in the endpoint file that this sending of issues from here is a deliberate hack with a date by which said hack should be removed

Reply:
that makes sense. a clean PR is coming